doc/cygnus-ngsi/integration/orion_cygnus_spark.md

```diff
@@ -0,0 +1,216 @@
+```
+
+In order to test the integracion among Orion Context Broker and Cygnus, simply update any context entity and check the related notification reception in Cygnus logs:
+
+```
+$ tail -f /var/log/cygnus/cygnus.log
+...
+time=2017-01-27T13:33:28.236UTC | lvl=INFO | corr=a72db6ff-ce15-4d38-a3a3-95af2ecc73b8 | trans=a72db6ff-ce15-4d38-a3a3-95af2ecc73b8 | srv=sc_valencia | subsrv=/gardens | comp=cygnus-ngsi | op=getEvents | msg=com.telefonica.iot.cygnus.handlers.NGSIRestHandler[304] : [NGSIRestHandler] Received data ({  "subscriptionId" : "51c0ac9ed714fb3b37d7d5a8",  "originator" : "localhost",  "contextResponses" : [    {      "contextElement" : {        "attributes" : [          {            "name" : "temperature",            "type" : "centigrade",            "value" : "26.5"          }        ],        "type" : "room",        "isPattern" : "false",        "id" : "room1"      },      "statusCode" : {        "code" : "200",        "reasonPhrase" : "OK"      }    }  ]})
+...
+```
+
+[Top](#top)
+
+##<a name="section4"></a>Setting up spark Streaming
+Apache Spark will be installed from the URL resulting from entering at [Spark Download page](http://spark.apache.org/downloads.html) and selecting:
+
+* Spark release: 1.6.3
+* Package type: pre-built for Hadoop 2.3
+* Download type: direct download
+
+The URL the "download Spark" link points to must be copied and then downloaded in our machine (into our home dir, for instance) by doing:
+
+```
+$ cd ~
+$ wget http://d3kbcqa49mib13.cloudfront.net/spark-1.6.3-bin-hadoop2.3.tgz
+--2017-01-18 14:40:31--  http://d3kbcqa49mib13.cloudfront.net/spark-1.6.3-bin-hadoop2.3.tgz
+Resolviendo d3kbcqa49mib13.cloudfront.net (d3kbcqa49mib13.cloudfront.net)... 54.240.186.114, 54.240.186.149, 54.240.186.159, ...
+Conectando con d3kbcqa49mib13.cloudfront.net (d3kbcqa49mib13.cloudfront.net)[54.240.186.114]:80... conectado.
+Petición HTTP enviada, esperando respuesta... 200 OK
+Longitud: 273211089 (261M) [application/x-tar]
+Grabando a: “spark-1.6.3-bin-hadoop2.3.tgz”
+
+100%[===========================================================================================================================================>] 273.211.089 30,8MB/s   en 9,6s   
+
+2017-01-18 14:40:40 (27,3 MB/s) - “spark-1.6.3-bin-hadoop2.3.tgz” guardado [273211089/273211089]
+```
+
+The downloaded file must be uncompressed and the removed:
+
+```
+$ tar xzf spark-1.6.3-bin-hadoop2.3.tgz
+$ rm spark-1.6.3-bin-hadoop2.3.tgz
+```
+
+Now, Spark should be available under our home dir, thus let's change directory:
+
+```
+$ cd spark-1.6.3-bin-hadoop2.3
+```
+
+[Top](#top)
+
+##<a name="section5"></a>Testing everything together
+In order to test the whole architecture of our real time context information analysis we'll use an already developed analysis application Spark provides within its examples, called `JavaFlumeEventCount`:
+
+```
+$ ls examples/src/main/java/org/apache/spark/examples/streaming/ | \
```

Comment:
Different from google doc, in the case of github, probably all the ls + grep fits in one line. No need for `\`